packages_rs/nextclade/src/tree/tree_attach_new_nodes.rs

```diff
@@ -52,6 +52,9 @@ fn add_aux_node(node: &mut AuspiceTreeNode) {
 
   let mut aux_node = node.clone();
   aux_node.branch_attrs.mutations.clear();
+  // Remove other branch attrs like labels to prevent duplication
+  aux_node.branch_attrs.other = serde_json::Value::default();
```

Comment:
This removes all attributes that are not explicitly listed in the [`TreeBranchAttrs` struct](https://github.com/nextstrain/nextclade/blob/a171903b93999dca5ef12ef01ff1a869b924ae31/packages_rs/nextclade/src/tree/tree.rs#L37-L43). What if users would like to keep some of the fields they added? Should we instead remove only a specific field or fields?

It can be done as follows:

 - Add optional field or fields to the struct, and mark them "skip if `None`":

   ```rust
   #[serde(skip_serializing_if = "Option::is_none")]
   pub field_name: Option<String>
   ```

 - Set these fields to `None` when needed. They will be then skipped during deserialization.

Reply:
Good question.

The tip is on a zero length branch, so all branch attrs I can think of should be on the actual branch leading to the new internal node.

That branch has moved so should all the labels, I'd say. Can't think of any labels we use that should be on both sides.

I'd say if the need arises we can still do the implementation you suggest above, but right now it'd not add anything.